.github/workflows/testsuite.yml

```diff
@@ -36,18 +36,15 @@ jobs:
     strategy:
       matrix:
         meteorRelease:
-          - '2.3'
-          - '2.8.0'
-          - '2.14'
-          # Latest version
+          - '3.0-beta.0'
```

Comment:
It's best to use the latest RC instead of the beta. This goes for all places that a meteor version is used.